docs/source/Synthesis.ipynb

```diff
@@ -0,0 +1,180 @@
+    "In addition, circuits have to be *mapped* to the target quantum device to satisfy its connectivity constraints.\n",
+    "Even though Clifford circuits form a finite subgroup of all quantum circuits -- one that is not even universal for quantum computing -- the search space for these problems grows exponentially with respect to the number of considered qubits.\n",
+    "\n",
+    "The *Clifford synthesis approach* in QMAP can be used to produce optimal Clifford circuits based on the methods proposed in <cite data-cite=\"schneider2023satEncodingOptimalClifford\">A SAT Encoding for Optimal Clifford Circuit Synthesis</cite>.\n",
+    "To this end, it encodes the underlying task as a satisfiability (SAT) problem and solves it using the [SMT solver Z3](https://github.com/Z3Prover/z3) in conjunction with a binary search scheme.\n",
+    "\n",
+    "The following gives a brief overview on Clifford circuits and how QMAP can be used for their synthesis."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Clifford Circuits\n",
+    "\n",
+    "Clifford circuits, i.e., circuits generated from the set $\\{H, S, \\mathit{CNOT}\\}$, form an important subclass of quantum circuits.\n",
+    "This is due to several factors\n",
+    "\n",
+    "- According to the Gottesman-Knill theorem, they can be simulated in polynomial time and space on classical computers using the *stabilizer* formalism.\n",
+    "- They can be used to describe several quantum phenomena such as superposition, entanglement, superdense coding, and teleportation.\n",
+    "- Many error correcting codes rely on them.\n",
+    "\n",
+    "Quantum states that can be obtained from the all-zero basis state $|0\\dots 0\\rangle$ by applying Clifford operations are called stabilizer states.\n",
+    "The name originates from the fact that such a state is uniquely and efficiently described by the set of operators that generate the group of its stabilizers.\n",
+    "Specifically, any *n*-qubit stabilizer state can be described by a set of *n* Pauli strings $\\pm P_{i,0}P_{i,1}P_{i,2}\\dots P_{i,n-1}$, with $P_{i,j}\\in\\{I, X, Y, Z\\}$ and $i, j\\in 0,\\dots, n-1$.\n",
+    "Hence,two bits per qubit are needed to identify the Pauli operator, as well as one additional bit for the phase, which leads to a total of $n(2n+1)` bits needed to uniquely describe a particular stabilizer state.\n",
+    "\n",
+    "The stabilizer representation of a quantum state is conveniently described by a *tableau*:\n",
+    "\n",
+    "$$\n",
+    "\n",
+    "    \\begin{bmatrix}\n",
+    "        x_{0,0}   & \\cdots & x_{0,n-1}   & z_{0,0}    & \\cdots & z_{0,n-1}   & r_0    \\\\\n",
+    "        \\vdots    & \\ddots &  \\vdots         & \\vdots     & \\ddots &    \\vdots         & \\vdots \\\\\n",
+    "        x_{n-1,0} & \\cdots & x_{n-1,n-1} & z_{n-1,0} & \\cdots & z_{n-1,n-1} & r_{n-1}  \\\\\n",
+    "    \\end{bmatrix}\n",
```

Comment:
This environment is broken in the documentation and not displayed properly.

Reply:
Locally it seems to work fine, but i'll have a look

Reply:
<img width="717" alt="image" src="https://user-images.githubusercontent.com/6358767/207910990-aeaa992a-9673-45a7-8f9a-810395a57c14.png">
still broken. (you can always check the current version for the PR if you click on the "Details" link for the corresponding CI check)

Reply:
This should be working now, there was an extra "\n" in there for some reason